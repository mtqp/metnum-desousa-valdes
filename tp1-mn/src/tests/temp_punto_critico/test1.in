400 400 100 50 400 1
300.0 150.0