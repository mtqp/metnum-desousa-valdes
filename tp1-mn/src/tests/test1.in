100 100 5 10 500 1 
30.0 20.0
