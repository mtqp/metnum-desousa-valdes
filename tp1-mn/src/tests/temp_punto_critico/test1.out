0	0	-100.00000
0	1	-100.00000
0	2	-100.00000
0	3	-100.00000
0	4	-100.00000
1	0	-100.00000
1	1	-29.14618
1	2	98.25659
1	3	400.00000
1	4	-100.00000
2	0	-100.00000
2	1	-14.84131
2	2	122.17255
2	3	400.00000
2	4	-100.00000
3	0	-100.00000
3	1	-52.39160
3	2	5.27492
3	3	51.31873
3	4	-100.00000
4	0	-100.00000
4	1	-100.00000
4	2	-100.00000
4	3	-100.00000
4	4	-100.00000
